API v1.1 (work in progress) (#30)
clarify/clarify-go: CONTRIBUTING.md

# Contributing

> This document is mainly aimed at audience within Searis AS, and describe information about how to contribute to this repository.

Reviewer: ```suggestion
This document is mainly aimed at audience within Searis AS, and describes information about how to contribute to this repository.
```

## Configure git

After first check out, run the following command to use our recommended git hooks:

```sh
git config core.hooksPath .githooks
```

Also ensure that the git author configuration correctly, using your _work email_, or an email (if you contribute on behalf of a company). See the [Git set-up guide](https://docs.github.com/en/get-started/quickstart/set-up-git#setting-up-git) from GitHub for details. We also recommend that you set up commit verification.

## Updating copyright

> When ever updating a file, ensure that the Copyright notice is up-to-date. I.e. if the the notice says "Copyright 2022 Searis AS", and the current year is "2028", then update the notice to "Copyright 2022-2028 Searis AS".

Reviewer: ```suggestion
Whenever updating a file, ensure that the Copyright notice is up-to-date. I.e. if the notice says "Copyright 2022 Searis AS", and the current year is "2028", then update the notice to "Copyright 2022-2028 Searis AS".
```

If you are not making the contribution on behalf of Searis AS, you should add a new copyright line instead:

```
// Copyright 2028 <Your company> <contact email for legal issues>
// Copyright 2020 Searis AS
```

The copyright notice containing the current year always needs to be on top.

## Atomic commits

> Avoid unrelated changes within a commit, and retain linear history. This require good knowledge of the Git rebase feature.

Reviewer: ```suggestion
Avoid unrelated changes within a commit, and retain linear history. This requires good knowledge of the Git rebase feature.
```
